config/server/email-config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  auth: {
+      user: 'theultrarangers@gmail.com',
+      pass: '@ultrarangers00'
+  },
+  service: 'gmail'
+}
```

Comment:
I've been thinking to centralized the config for the server, something like config.json config.development.json etc. And we will have a config.service to fetch the config we want. So that we will hand rolled the requiring the specific. If that is ok for you let's do that on some other time.

Reply:
Let me think of it but yeah that would be a good thing to have

Reply:
I've added the `configService` and used it in the `emailService`.